Source/WebInspectorUI/UserInterface/Views/FontVariationDetailsSectionRow.js

```diff
@@ -47,30 +54,40 @@ WI.FontVariationDetailsSectionRow = class FontVariationDetailsSectionRow extends
         this._inputRangeElement.name = tag;
         this._inputRangeElement.min = minimumValue;
         this._inputRangeElement.max = maximumValue;
-        this._inputRangeElement.value = value ?? defaultValue;
-        this._inputRangeElement.step = this._getAxisResolution(minimumValue, maximumValue, tag);
+        this._inputRangeElement.step = this._step;
 
         this._variationMinValueElement = this._variationRangeElement.appendChild(document.createElement("div"));
         this._variationMinValueElement.className = "variation-minvalue";
         this._variationMinValueElement.textContent = this._formatAxisValueAsString(minimumValue);
+        this._variationMinValueElement.tooltip = WI.UIString("Minimum value of variation axis", "Min axis value @ Font Details Sidebar");
 
         this._variationMaxValueElement = this._variationRangeElement.appendChild(document.createElement("div"));
         this._variationMaxValueElement.className = "variation-maxvalue";
         this._variationMaxValueElement.textContent = this._formatAxisValueAsString(maximumValue);
+        this._variationMinValueElement.tooltip = WI.UIString("Maximum value of variation axis", "Max axis value @ Font Details Sidebar");
+
+        this._valueInputElement = this.element.appendChild(document.createElement("input"));
+        this._valueInputElement.type = "number";
+        this._valueInputElement.name = tag;
+        this._valueInputElement.step = this._step;
+        this._valueInputElement.min = this._minimumValue;
+        this._valueInputElement.max = this._maximumValue;
+
+        this._valueInputElement.addEventListener("keydown", this._handleValueKeydownEvent.bind(this), {signal: abortSignal});
+        this._valueInputElement.addEventListener("input", this._handleValueInputEvent.bind(this), {signal: abortSignal});
+        this._valueInputElement.addEventListener("blur", this._handleValueBlurEvent.bind(this), {signal: abortSignal});
 
         this._inputRangeElement.addEventListener("input", (event) => {
-            this.value = event.target.value;
-            this.dispatchEventToListeners(WI.FontVariationDetailsSectionRow.Event.VariationValueChanged, {tag: event.target.name, value: event.target.value});
+            this.value = event.target.valueAsNumber;
+            this.dispatchEventToListeners(WI.FontVariationDetailsSectionRow.Event.VariationValueChanged, {tag: this._tag, value: this._value});
```

Comment:
Since we are doing `_handle` methods for the text field, this might be a good time to turn this into its own method as well so that all the input handling code is in the same general place below.